tests/sentry/seer/anomaly_detection/test_get_anomaly_data.py

```diff
@@ -0,0 +1,92 @@
+from unittest.mock import MagicMock, Mock, patch
+
+from urllib3.exceptions import TimeoutError
+from urllib3.response import HTTPResponse
+
+from sentry.seer.anomaly_detection.get_anomaly_data import get_anomaly_threshold_data_from_seer
+from sentry.snuba.models import QuerySubscription
+from sentry.testutils.cases import TestCase
+
+
+class GetAnomalyThresholdDataFromSeerTest(TestCase):
+    def setUp(self) -> None:
+        super().setUp()
+        subscription = self.create_alert_rule(
```

Comment:
Same comments as in the other test - you can create the subscription without creating an alert rule